src/test/suite/features/iam/createProjectProcess.test.ts

```diff
@@ -10,7 +10,7 @@ import { authHelper } from '../../../../auth/authHelper'
 describe('createProjectProcess()', () => {
   const projectName = 'fake-project-name'
   const consoleAuthToken = 'fake-console-auth-token'
-  const projectId = 'fake-project-id'
+  const projectId = 'fake-project-id-createProjectProcess'
```

Comment:
I had a slightly different idea, which might be a little bit easier to maintain (and will prevent people from accidentally forgetting about it):
```ts
// helpers.ts
import crypto from 'crypto'

export function generateProjectId() {
  return `project-id-${crypto.randomBytes(10).toString('hex')}`
}
```

```suggestion
  const projectId = generateProjectId()
```

We use similar approach in some other repos too.

Reply:
I thought about this, just don't want to spend too much time
but will check